<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMember</class>
 <widget class="QDialog" name="AddMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加成员</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { 
	font-size: 10pt; 
	padding: 5px; 
	border: 1px solid #b2b2b2; 
	border-radius: 5px; 
	color: #333; 
}

QLabel
{
	border : 1.2px solid black;
	padding: 5px; 
	border-radius: 5px;
}

QComboBox
{
	font-size: 10pt; 
	padding: 5px; 
	border: 1px solid #b2b2b2; 
	border-radius: 5px; 
	color: #333; 
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="9,1">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="modifyLayout" stretch="0,0,0">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="commitPushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="emailLable">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>邮箱</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailLineEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="majorLable">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>专业</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="majorLineEdit">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="deptLable">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>系别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="deptLineEdit">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="genderLable">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="genderComboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>沃尔玛购物袋</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>大润发购物袋</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="stuidLabel">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>学号</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stuidLineEdit">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="nameLable">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLineEdit">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="ageLable">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>年龄</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ageLineEdit">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="phoneLable">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>联系方式</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneLineEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
